packages/react-html/README.md

```diff
@@ -230,6 +230,18 @@ Renders a `<link />` tag with the necessary props to specify a favicon. Accepts
 
 Renders a `<link />` tag that preconnects the browser to the host specified by the `source` prop. You can read more about preconnecting on [Google’s guide to resource prioritization](https://developers.google.com/web/fundamentals/performance/resource-prioritization#preconnect).
 
+### `<Baseline />`
+
+Renders a minimal set of `<Meta />` tags to specify basic app information. Accepts a `title` property that is passed as the children of a `<Title />` component, and a `favicon` source that is passed to a `<Favicon />` component.
+
+### `<Responsive />`
```

Comment:
I'm also not sure we ever don't want these

Reply:
This one I think is good, the incantation to get the right meta tags is always a pain in the but IMO

Reply:
Yeah, I just mean do we ever want to not have this? Why not just have the component always include them